backend-tests/docker/docker-compose.backend-tests.yml

```diff
@@ -8,10 +8,15 @@ services:
             - ${INTEGRATION_PATH}/backend-tests/tests:/tests
             - ${INTEGRATION_PATH}/testutils:/testutils
             - /var/run/docker.sock:/var/run/docker.sock
-        depends_on:
-            - mender-api-gateway
+            - ~/.aws:/root/.aws
+            - ~/.kube:/root/.kube
```

Comment:
The test runs in the docker container, so I have to "log in" to kubernetes (and aws, if we are using the iam authenticator) from there; locally, I'm already authenticated so exporting those folders is enough to make kubectl work in the container. I can imagine we are doing the same in gitlab (log in from the job executor worker, then export those folders to the container) or if this is an issue, we can remove those volume mounts and log in from the docker container. The current approach is very developer friendly, because you can run the tests locally without configuring any authentication mechanism inside the container.

Reply:
> I can imagine we are doing the same in gitlab (log in from the job executor worker, then export those folders to the container)

That was my thinking too :+1: